Feature/pace loading

CyrilCharlier opened the pull request:
Changement du loader : Passage au Pace d'AdminLte

Ajout du JS du plugin
Ajout du SCSS
Modification de la page de Customisation
Suppression de la référence de cette préférence dans la Back (Fichier Ini) et Front (JS)

PR lié à l'issue #1400

CyrilCharlier committed:
Changement Barre de Chargement

Remplacement du Loading initial par le Pace Loading

CyrilCharlier committed:
Merge remote-tracking branch 'origin/develop' into feature/PaceLoading

CyrilCharlier committed:
Nettoyage Front/Back ancien spinner

Nettoyage des traces de l'ancien spinner au profit du nouveau

CyrilCharlier committed:
Prise en compte nouveau SCSS

Prise en compte du nouveau fichier SCSS dans nextdom.scss

ColonelMoutarde reviewed (changes requested):
Salut,
Il faut que tu utilise les fonctionnalité de sass stp. Là c'est du code css.
Peux-tu refactorer stp

CyrilCharlier commented:
euh oui mais là il faudrait un coup de main car je n'y connais rien

CyrilCharlier commented:
@ColonelMoutarde c'est du SCSS ou du sass ?
Car après quelques recherches ce serait plutôt du SCSS ?

Je ne voudrais pas partir sur un mauvais refactoring. 

Merci et bonne journée

ColonelMoutarde commented:
Salut, Sass est le nom du protocole. On peut faire du sass et du scss. Nous utilisons scss.

CyrilCharlier commented:
Salut, comme tu vois je ne connais vraiment pas :D Merci pour la précision

byackee committed:
use npm for pace-js

CyrilCharlier committed:
Migration CSS dans le Theme

Suite au conseil de @cyrilphoenix71 , passage du code CSS dans le theme pour prendre en compte la color de contraste du theme

CyrilCharlier committed:
Purges des fonctios *LoadingCustom et SCSS

Purge des deux fonctions *LoadingCustom
Remise en place du _pace.scss
background du pace dans le theme.html

CyrilCharlier commented:
Voilà les différentes remarques ont été prises en compte. Si vous en avez d'autres, n'hésitez pas à me le dire

CyrilCharlier committed:
Update theme.html.twig

Regroupement du CSS dans la section COLOR2 = Couleur de contraste

cyrilphoenix71 committed:
fix pace position on mobile resolution
